src/sentences.rs

```diff
@@ -26,6 +30,11 @@ pub struct Sentence {
     pub structure: Option<String>,
 }
 
+#[derive(FromForm)]
+struct LastId {
+    pub id: UUID,
+}
```

Comment:
we need this because we can't implement a trait (FromForm) on a type ( UUID) that don't belong to us ?

Reply:
if so, it should be an issue raised to the rocket_contrib/uuid crate , as this crate is specifically made to ease interaction with uuid in rocket, so it should do that for us.

Reply:
I'm gonna do a second check on this point.